package.json

```diff
@@ -26,6 +26,7 @@
     "next": "^12.0.2",
     "next-seo": "^4.28.1",
     "react": "^17.0.2",
+    "react-chartjs-2": "^3.3.0",
```

Comment:
Is this still having charts 
I thought you said this is now segmented from the graphs work?

Reply:
- [x] remove package